subprojects/docs/src/docs/userguide/authoring-builds/authoring_maintainable_build_scripts.adoc

```diff
@@ -104,7 +105,11 @@ org/gradle/model/**
 org/gradle/testkit/**
 org/gradle/testing/**
 org/gradle/vcs/**
+org/gradle/work/**
 org/gradle/workers/**
+org/gradle/util/**
+org/gradle/kotlin/dsl/*
+org/gradle/kotlin/dsl/precompile/*
```

Comment:
🤔 Shall we try and keep this ordered alphabetically?

Reply:
I wouldn't insert Kotlin DSL entries into the original public API list. It could create confusion when updating the first list.

Right now (after this PR), there is one-to-one line correspondence between the lists, which is nice, because one can check they are the same at least by doing something like "Compare with clipboard".

What could be better here, maybe, is splitting them in two code snippets? It would be easier to understand, that each list is sorted alphabetically.

Reply:
I split them in two snippets